src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_ja.properties

```diff
@@ -66,6 +66,7 @@ unnecessary.paren.string=文字列 {0} の前後に不要な括弧がありま
 unnecessary.semicolon=不要なセミコロン。
 unused.catch.parameter=未使用のキャッチパラメータ ''{0}'' には名前を付けないでください。
 unused.lambda.parameter=未使用のラムダパラメータ ''{0}'' には名前を付けないでください。
+unused.local.method=ローカル変数 '' {0} '' が未使用です。
```

Comment:
c&p as i cant edit this. Who to fix this plz?

Reply:
@nrmancuso could you help me out on this one please? its a new feature which we all like and its kind of approved, even if not formally. Its just this little issue, then we can work this out pretty quick.